test/network_test.cc

```diff
@@ -19,9 +21,12 @@ TEST(NetworkTest, MessageBrokerTest) {
       ex_actor::internal::util::SetThreadName("node_" + std::to_string(node_id));
       ex_actor::internal::network::MessageBroker message_broker(
           node_list,
-          /*this_node_id=*/node_id, [&message_broker](uint64_t receive_request_id, ByteBufferType data) {
+          /*this_node_id=*/node_id,
+          [&message_broker](uint64_t receive_request_id, ByteBufferType data) {
             message_broker.ReplyRequest(receive_request_id, std::move(data));
-          });
+          },
+          HeartbeatConfig {.heartbeat_timeout = std::chrono::milliseconds(2000),
```

Comment:
One last thing, can you create a new header to contain all this constants(e.g. constatns.h)? So we don't need to write them here and there anymore.

And also provide a default value in MessageBroker's constructor.